Pre-Processing/get_non_soft_story_images.py

```diff
@@ -39,14 +46,23 @@ def get_image(address):
 
```

Comment:
It would be better to put everything in a function. This makes your code more modular and reusable. 

If you want to call the script, you do:

```
if __name__ == "__main__":
```